.github/ISSUE_TEMPLATE/release-tracking-issue.md

```diff
@@ -24,30 +25,28 @@ with the release so please keep this issue up to date:
   appears in the checkboxes.
 - Add any updates that may be of interest to the community (such as delays) as
   comments on this issue, including after the release is promoted to Stable (and, in some cases, LTS).
-- Keep the title of the issue updated to reflect whether this issue is tracking
-  the Experimental/Beta builds or the build in Stable.
 
-Note: remove the backticks (``) from the link.
 -->
 
-- [x] Release `[<RELEASE_NUMBER>](https://github.com/ampproject/amphtml/releases/tag/<RELEASE_NUMBER>)` is cut as a new canary release
+This issue tracks release [<RELEASE_NUMBER>](https://github.com/ampproject/amphtml/releases/tag/<RELEASE_NUMBER>)
+
 - [ ] Release <RELEASE_NUMBER> promoted to Experimental and Beta (opt-in) channels (<CL submit time>)
 - [ ] Release <RELEASE_NUMBER> promoted to Experimental and Beta (1% traffic) channels (<CL submit time>)
 - [ ] Release <RELEASE_NUMBER> promoted to Stable channel (<CL submit time>)
 
 <!--
-On the second Monday of each month, the current stable version will be promoted to the LTS release channel. In other words: 
+On the second Monday of each month, the current Stable version will be promoted to the LTS release channel. In other words:
 
-- If this release is promoted to stable on the first Tuesday of a given month, it needs to be promoted to LTS on the second Monday of the same month.
-- If this release is promoted to stable on the second, third, fourth, or fifth Monday of a given month, it is not an LTS release candidate (but will be included in a later LTS release).
+- If this release is promoted to Stable on the first Tuesday of a given month, it needs to be promoted to LTS on the second Monday of the same month.
+- If this release is promoted to Stable on the second, third, fourth, or fifth Monday of a given month, it is not an LTS release candidate (but will be included in a later LTS release).
 
 Based on the above, if this release must be promoted to LTS, copy-paste the following checkbox into the list above.
 
 - [ ] Release <RELEASE_NUMBER> promoted to LTS (<CL submit time>)
 
 If you perform cherry picks, add/update the checkboxes above as needed e.g.
 
-- [ ] Release `[<CHERRY_PICK_RELEASE_NUMBER>](...)` created with cherry picks
+- [ ] Release [<CHERRY_PICK_RELEASE_NUMBER>](...) created with cherry picks
```

Comment:
I'd go a step further and pre-fill this with the incomplete URL.

```markdown
[ ] Release [<CHERRY_PICK_RELEASE_NUMBER>](https://github.com/ampproject/amphtml/releases/tag/<CHERRY_PICK_RELEASE_NUMBER>) created with cherry picks
```

Reply:
Done + added a placeholder to add the cherry pick issue # number